src/domain/errorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
The left menu should be not visible in the error page if you check the layout specified in Abstract.

Reply:
Fixed.